_layouts/post.html

```diff
@@ -37,11 +37,20 @@ <h1 id="{{ page.title | cgi_escape }}" class="title">{{ page.title }}</h1>
 {% include blog/post_nav.html %}
 {% endif %}
 
+<!--Utterances-->
+{% if site.comments.utterances.repo %}
+{% if site.comments.utterances.issue-term %}
+{% if site.comments.utterances.theme %}
+{% include social/utterances.html %}
+{% endif %}
+{% endif %}
+{% endif %}
```

Comment:
Can they all be in one if like the other ones

Reply:
Look, the point here is that, if repo is filled and issue term isn't filled, utterances won't load. Similarly, if issue term and repo is  filled but theme is not, Utterances won't work. The point here is that, since `repo`,`issue-term`, and `theme` are required, if they are full in `config.yml`, only them Utterances will load on the site.

Reply:
no

Reply:
@coder-lg I was think of something like:

```
{% if site.comments.utterances.repo and site.comments.utterances.issue-term and site.comments.utterances.theme %} {% include social/utterances.html %} {% endif %}
```

Since they are all needed, let's use a `and` in the if 🙂

Reply:
wow
copy that

Reply:
done

Reply:
we need 3 or more but the first three are cumpolsory, label is optional